Lib/logging/handlers.py

```diff
@@ -1399,7 +1399,7 @@ def flush(self):
         records to the target, if there is one. Override if you want
         different behaviour.
 
-        The record buffer is also cleared by this operation.
+        The record buffer is only cleared afterwards, when target was set.
```

Comment:
I think I would express this as:  "The record buffer is only cleared if a target has been set."